### PR TITLE
LPS-126652 | master

### PR DESCRIPTION
**Pull request by joepak**
[LPS-126652](https://issues.liferay.com/browse/LPS-126652)

QA Review Needed
@david-gutierrez-mesa
@Tim-Cao
@liulukiwi

### CODE REVIEW

**Comment by joepak** [comment]
ci:test:sf

**Comment by joepak** [comment]
ci:test:revelant

**Comment by joepak** [comment]
ci:test:sf

**Comment by joepak** [comment]
ci:test:relevant;nocompile

**Comment by Tim-Cao** [comment]
ci:test:relevant:nocompile

**Comment by Tim-Cao** [comment]
Just start reviewing

**Comment by Tim-Cao** [comment]
ci:test:relevant

**Comment by joepak** [comment]
ci:test:sf

**Comment by joepak** [comment]
ci:test:relevant;nocompile

**Comment by joepak** [comment]
@Tim-Cao just added a commit to fix unrelated SF error

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase [hunk 1]
```diff
@@ -12,6 +12,25 @@ definition {
 
 		JSONGroup.addGroup(groupName = "Test Site Name");
 
+		JSONLayout.addPublicLayout(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase:15]: Should enable Local Staging prior to conversion. 
Move line 15-32 into case from setup

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase [hunk 2]
```diff
@@ -34,8 +53,20 @@ definition {
 	test ViewContentPageInLiveAfterConvertingAndPublishingWidgetPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase:53]: Missing steps to view content page in Live site.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase [hunk 3]
```diff
@@ -34,8 +53,20 @@ definition {
...
 		property portal.acceptance = "false";
 
-		// TODO LPS-98327 ViewContentPageInLiveAfterConvertingAndPublishingWidgetPage pending implementation
+		ProductMenu.gotoPortlet(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase:56]: Line 56-58 are unnecessary

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase [hunk 4]
```diff
@@ -34,8 +53,20 @@ definition {
...
+
+		Button.clickPublish();
 
+		AssertElementPresent(
+			key_pageName = "Test Page Name",
+			key_pageType = "Content Page",
+			locator1 = "SitePages#LIST_GROUP_ITEM_PAGE_TYPE");
 	}
 
 	@description = "This is a test for LPS-98327. When a widget page is converted to a content page and the content page is published, the content page has the original plid identifier."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase:72]: This description should be **When a widget page is converted to a content page and the content page is not published yet, the content page has the original plid identifier.**

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase [hunk 5]
```diff
@@ -34,8 +53,20 @@ definition {
...
+			category = "Site Builder",
+			portlet = "Pages");
+
+		SitePages.gotoPageMenuItem(
+			menuItem = "Edit Conversion Draft",
+			pageName = "Test Page Name");
+
+		Button.clickPublish();
 
+		AssertElementPresent(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverterWithStaging.testcase:66]: Should check page type in both Staging site and Live site. Additionally, assert the Asset Pubisher is shown in page.

**Comment by Tim-Cao** [comment]
Hi @joepak, I just left my comments, feel free to ask. Btw, thanks for writing.

**Comment by joepak** [comment]
Hi @Tim-Cao, I created a new pr based on your edits. 
Please see: https://github.com/liferay-echo/liferay-portal/pull/3541

**Comment by Tim-Cao** [comment]
Thanks @joepak

**Pull request closed without merging**